packages/dapp/src/contexts/Web3Context.jsx

```diff
@@ -65,6 +71,28 @@ const providerOptions = {
     options: { rpc },
     connector,
   },
+  'custom-walletlink': {
+    display: {
+      logo: coinbaseLogo,
+      name: 'Coinbase',
+      description: 'Scan with WalletLink to connect',
+    },
+    options: {
+      appName: 'OmniBridge',
+      networkUrl: getRPCUrl(homeChainId),
+      chainId: homeChainId,
+    },
+    package: WalletLink,
+    connector: async (_, options) => {
+      const { appName, networkUrl, chainId } = options;
+      const walletLink = new WalletLink({
+        appName,
+      });
+      const provider = walletLink.makeWeb3Provider(networkUrl, chainId);
```

Comment:
```suggestion
      const provider = walletLink.makeWeb3Provider({}, 0);
```
This makes walletLink generic and allows any network without any issues.

Reply:
With this change we can remove import of `networks` of `DEFAULT_BRIDGE_NETWORK` etc

Reply:
Alright. Changed it now.